students/rmart300/trigram/trigram.py

```diff
@@ -0,0 +1,45 @@
+
+trigram_dict={}
+
+def read_book(book_file):
+
+    book = open(book_file, 'r'):
+    digram = ''
+    for line in book:
+        line.strip()
+        words = line.split()
+        for i in range(len(words)):
+            if i > 1:
+                digram = f"{words[i-1]} {words[i]}"
+            if i < len(words)-1:
+                if digram in triagram_dict:
+                    word_list = trigram_dict[digram]
+                    word_list.append(words[i+1])
+                    trigram_dict[digram] = wordlist
```

Comment:
note that you can simplify lines 16-18 to: `trigram_dict[digram].append(words[i + 1])`